.idea/scopes/prettyPrinters.xml

```diff
@@ -0,0 +1,3 @@
+<component name="DependencyValidationManager">
+    <scope name="prettyPrinters" pattern="file[intellij-rust]:prettyPrinters/*.py*" />
```

Comment:
Looks like it doesn't work properly - I still see weak warnings in `lldb/__init__.pyi`
![image](https://user-images.githubusercontent.com/2539310/104204492-65f20f00-543e-11eb-9af6-3484aba0167d.png)

Reply:
Well, I'm not sure why it doesn't work and how to fix it actually. For me, it works fine even when I check out this branch on another machine.

Could you try to open `.idea/inspectionProfiles/intellij_rust.xml` replace
`<scope name="prettyPrinters" level="INFORMATION" enabled="true" />` with
`<scope name="prettyPrinters" level="WEAK WARNING" enabled="false" />`?